INFO     cardano_node_tests.tests.conftest:conftest.py:135 Changed CWD to '/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-17/popen-gw1'.
INFO     cardano_node_tests.utils.cluster_management:cluster_management.py:414 Stopping cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-17/cluster5/startup_files/pcpfkfhw/stop-cluster-hfc`.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:340 Starting cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-17/cluster5/startup_files/pcpfkfhw/start-cluster-hfc`.
ERROR    cardano_node_tests.utils.cluster_management:cluster_management.py:437 Failed to start cluster: An error occurred while running `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-17/cluster5/startup_files/pcpfkfhw/start-cluster-hfc`: Retrying `cardano-cli byron transaction submit-tx --testnet-magic 42 --tx /home/martink/Source/repos/cardano-node5/state-cluster5/byron/tx1.tx`. Failure:
MuxError (MuxIOException writev: resource vanished (Broken pipe)) "(sendAll errored)"

cardano-cli: MuxIOException writev: resource vanished (Broken pipe) "(sendAll errored)"
Retrying `cardano-cli byron transaction submit-tx --testnet-magic 42 --tx /home/martink/Source/repos/cardano-node5/state-cluster5/byron/tx1.tx`. Failure:
MuxError (MuxIOException writev: resource vanished (Broken pipe)) "(sendAll errored)"

cardano-cli: MuxIOException writev: resource vanished (Broken pipe) "(sendAll errored)"
Retrying `cardano-cli byron transaction submit-tx --testnet-magic 42 --tx /home/martink/Source/repos/cardano-node5/state-cluster5/byron/tx1.tx`. Failure:
MuxError (MuxIOException writev: resource vanished (Broken pipe)) "(sendAll errored)"

cardano-cli: MuxIOException writev: resource vanished (Broken pipe) "(sendAll errored)"

INFO     cardano_node_tests.utils.cluster_management:cluster_management.py:414 Stopping cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-17/cluster5/startup_files/pcpfkfhw/stop-cluster-hfc`.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:340 Starting cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-17/cluster5/startup_files/pcpfkfhw/start-cluster-hfc`.
ERROR    cardano_node_tests.utils.cluster_management:cluster_management.py:437 Failed to start cluster: An error occurred while running `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-17/cluster5/startup_files/pcpfkfhw/start-cluster-hfc`: Retrying `cardano-cli byron transaction submit-tx --testnet-magic 42 --tx /home/martink/Source/repos/cardano-node5/state-cluster5/byron/tx1.tx`. Failure:
MuxError (MuxIOException writev: resource vanished (Broken pipe)) "(sendAll errored)"

cardano-cli: MuxIOException writev: resource vanished (Broken pipe) "(sendAll errored)"
Retrying `cardano-cli byron transaction submit-tx --testnet-magic 42 --tx /home/martink/Source/repos/cardano-node5/state-cluster5/byron/tx1.tx`. Failure:
MuxError (MuxIOException writev: resource vanished (Broken pipe)) "(sendAll errored)"

cardano-cli: MuxIOException writev: resource vanished (Broken pipe) "(sendAll errored)"
Retrying `cardano-cli byron transaction submit-tx --testnet-magic 42 --tx /home/martink/Source/repos/cardano-node5/state-cluster5/byron/tx1.tx`. Failure:
MuxError (MuxIOException writev: resource vanished (Broken pipe)) "(sendAll errored)"

cardano-cli: MuxIOException writev: resource vanished (Broken pipe) "(sendAll errored)"